manifest.json

```diff
@@ -18,17 +18,7 @@
   },
   "content_scripts": [
     {
-      "matches": [
-        "https://github.com/*",
-        "*://*/*.pu*",
-        "*://*/*.puml*",
-        "*://*/*.plantuml*",
-        "*://*/*.wsd*",
-        "file:///*/*.pu*",
-        "file:///*/*.puml*",
-        "file:///*/*.plantuml*",
-        "file:///*/*.wsd*"
-      ],
+      "matches": ["https://*/*", "http://*/*", "file:///*/*", "*://*/*"],
```

Comment:
`"*://*/*"` should be removed (We now consider only http, https and file scheme)